x-pack/packages/ai-infra/product-doc-artifact-builder/src/tasks/process_documents.ts

```diff
@@ -39,7 +39,7 @@ const removeDuplicates = (documents: ExtractedDocument[]): ExtractedDocument[] =
 const filterEmptyDocs = (documents: ExtractedDocument[]): ExtractedDocument[] => {
   return documents.filter((doc) => {
     const tokenCount = encode(doc.content_body).length;
-    if (tokenCount < 100) {
+    if (tokenCount < 120) {
```

Comment:
What does this magic number represent?

Reply:
Do you know Sonar's S109 😄 ? 

More seriously, that's an estimate of the limit between "empty" documentation page (we have a bunch of pages just displaying "this documentation was moved to XXX) and the rest.

I'll extract the magic number to a const and add a comment